src/plugins/intel_cpu/src/utils/serialize_base.hpp

```diff
@@ -3,36 +3,36 @@
 //
 #pragma once
 
-#include <functional>
-#include <iostream>
+#include <pugixml.hpp>
 
 #include "openvino/core/model.hpp"
-#include "openvino/util/mmap_object.hpp"
 
 namespace ov {
 namespace intel_cpu {
 
 class ModelSerializer {
 public:
     ModelSerializer(std::ostream& ostream);
+
     void operator<<(const std::shared_ptr<ov::Model>& model);
 
 private:
-    std::ostream& _ostream;
+    std::ostream& m_ostream;
 };
 
-class ModelDeserializer {
+class ModelDeserializerBase {
```

Comment:
Please, add virtual destructor to prevent any leaks

Reply:
Done